recipes-images/images/engicam-gwc.bb

```diff
@@ -98,6 +96,7 @@ IMAGE_INSTALL:append = " \
 	kernel-module-dm-crypt \
 	keyutils \
 	keyctl-caam \
+	caam-keyblob \
```

Comment:
Are you sure this is correct and needed? If you look above the recipe is `kernel-module-caam-keyblob` that should be enough to deliver the needed driver (in modules directory under `extra/caam_keyblob.ko`).

Reply:
Yes, show be enough! good catch